docs/reference/deprecated_api.rst

```diff
@@ -0,0 +1,22 @@
+====================
+Deprecated Endpoints
+====================
+
+.. XXX: As part of FLOC 3518 the Container API Endpoints below have been deprecated, as Flocker is not a container framework:
+
+.. warning:: 
+   The endpoints listed in this document are deprecated, and will be removed in a future version of Flocker.
+
+The Flocker Container API enables you to manage containers in your cluster.
+However, ClusterHQ is no longer developing any future features in this area, as you can now use Flocker with 3rd party orchestration tools.
+
+The deprecated endpoints listed below are still available for use, but please be aware that they will be removed in a future version of Flocker:
```

Comment:
Hmm!  Forward looking statements.  Oh my.  Always a bit out on a limb with these...

The philosophy behind the URL hierarchy used with these endpoints - specifically the `v1` prefix - is that as long as we provide these at all, we won't change how they work or what they do.  And if we're being really precise, I think that means that the whole collection of v1 endpoints should remain unchanged.  This probably means that what we're likely to do is introduce a `v2` which has the dataset endpoints but not the container endpoints.  Then we'll carry on with v1 and v2 for a while and then eventually we'll remove all of v1 - at which point, indeed, the container endpoints will be entirely unavailable.

However, we're very inexperienced in this area (we are at v1, after all - ie, we've never *actually* gone through such a transformation) so there's some chance when we actually go and try to do this we'll discover some reason it's a bad idea (or maybe even an infeasible idea) and have to do something else.

My best suggestion is to change the phrasing here to be something like "they will not be available in future versions of the Flocker API".  Pretty similar and maybe the subtlety will be lost on most of our audience without a more complex explanation of our compatibility policy and change workflow (which I suspect is currently missing from the docs) - so I could be making too much of this point - but also easy to do so maybe worth it.

Maybe @itamarst has some thoughts about this as well.